great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -101,8 +105,8 @@ def __init__(
         self,
         name: str,
         config_version: float,
-        rules: Dict[str, CommentedMap],
         variables: Optional[Dict[str, Any]] = None,
+        rules: Optional[Dict[str, Dict[str, Any]]] = None,
```

Comment:
Is this truly `Any`? I understand we want to be open-ended with our rule configs but can we tighten this up at all? Don't have a good solution here.

Reply:
@cdkini Here, it is a bit of a "lazy" Any (as in, all sorts of objects), but I feel it is OK to leave as is for now.

Reply:
That's fair - it's any primitive right? I think Any is much more concise than Union[str, int, float, list, dict]